### DISCUSSION TITLE
request to present

### DISCUSSION THREAD

**Discussion by 5shekel** [cables-gl/cables_docs#574] [Answers & Questions] [answered]
hey peeps, 
i was at ccc camp across from you guys at the analog synth hardware thingy.
we run a live coding interest group in our hackerspace.   
for the last 3-4 years twice a month on Thursdays. 
https://telavivmakers.org/music/Home

would love if one of you can demo cabels.gl and mybe do a short tutorial.  

i did talk with one of you present in the cmp but its all a blurrr :)
anyways if you into that let me know, thanks!

**Comment by steam0r** [comment, marked as answer]
hey,

i remember you, we talked a bit and you gave me your address. didnt get to contacting you yet, though. sorry for that.

totally up for it. let's talk! i think discord would work best (`steam5711`). would be cool to have some directions on
the audience, what to show, who to invite, lenght, when and where...but we can discuss this in person much better, i think.

looking forward to hearing from you!

best,
stephan/steam
